### DISCUSSION TITLE
useHover in mobile & docs

### DISCUSSION THREAD

**Discussion by KutnerUri** [floating-ui/floating-ui#2739] [React]
I am making sure that useHover() behaves correctly in mobile (which it does, as far as I can tell, I even tried [CodeSandbox hover example](https://codesandbox.io/p/sandbox/late-https-lu3833))

What I am missing is some reference in the docs, explaining how it behaves in mobile, something like:

```
### Mobile view

In touch devices, useHover() behaves the same as CSS :hover(), using onTouchStart and onTouchEnd events.
It will open the popup when touching the reference element,
and close it when touching other elements (but not the same element?)
```

Not sure about:
- What's the exact behavior? What events are used and why?
- How does it interact with useClick() in mobile?
- How does it interact with the safePolygon
- Maybe caveats or specific device/browser behavior?

**Comment by FezVrasta** [comment]
I don't think hover events should be used on touch screens. The mobile browsers bind the touch events to hover for compatibility purposes but it's not a good user experience for users.

**Reply by KutnerUri** [reply]
I think the whole point is that floating-ui handles the edge cases, like touch devices?

what would be the dev exp if hover won't apply to mobile devices? `useTouch()`?
